include/solvers/navier_stokes_base.h

```diff
@@ -474,6 +480,13 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   const unsigned int pressure_fem_degree;
   unsigned int       number_quadrature_points;
 
+  // Mappings and Quadratures
+  std::shared_ptr<Mapping<dim>>        velocity_mapping;
+  std::shared_ptr<Mapping<dim>>        pressure_mapping;
```

Comment:
Is it really pertinent to have two mappings? I find this a bit weird. I would always use the mapping for Velocity for both variables if it were me...

Reply:
I considered that since velocity and pressure degree wouldn't always be the same, they should have each their own mapping. Maybe I don't understand what is a mapping and we could use the mapping associated with variable of the highest degree? (so velocity)